### PR TITLE
TASK-2438

### PR DESCRIPTION
**Pull request by j-coll**
TASK-2438

### CODE REVIEW

**Comment by j-coll** [comment]
Task linked: [TASK-2438 EGLH Solid Cancer: to be able to filter by role in cancer independent if a panel is applied](https://app.clickup.com/t/36631768/TASK-2438)

**Review by jtarraga — APPROVED** [review]

File: opencga-core/src/main/java/org/opencb/opencga/core/common/VersionUtils.java [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+//            System.out.println(minVersion + "\t=\t" + version);
+//        } else if (c < 0) {
+//            System.out.println(minVersion + "\t<\t" + version);
+//        } else {
+//            System.out.println(minVersion + "\t>\t" + version);
+//        }
+        return c <= 0;
+    }
+
+//    public static void main(String[] args) {
```
Comment (pfurio) [inline opencga-core/src/main/java/org/opencb/opencga/core/common/VersionUtils.java:28]: This code should be removed and added as part of a junit test, right?

**Review by pfurio — APPROVED** [review]
I would remove the commented code I mentioned in my comment. Apart from that, everything is fine.

File: opencga-core/src/main/java/org/opencb/opencga/core/common/VersionUtils.java (hunk 1 above)
Comment (j-coll) [reply opencga-core/src/main/java/org/opencb/opencga/core/common/VersionUtils.java:28]: totally right

**Pull request merged**